ArduSub/mode_surface.cpp

```diff
@@ -32,28 +30,32 @@ void ModeSurface::run()
         return;
     }
 
-    // Already at surface, hold depth at surface
-    if (sub.ap.at_surface) {
-        set_mode(Mode::Number::ALT_HOLD, ModeReason::SURFACE_COMPLETE);
-    }
-
-    // convert pilot input to lean angles
-    // To-Do: convert sub.get_pilot_desired_lean_angles to return angles as floats
-    sub.get_pilot_desired_lean_angles(channel_roll->get_control_in(), channel_pitch->get_control_in(), target_roll, target_pitch, sub.aparm.angle_max);
+    if (sensorless_mode) {
```

Comment:
maybe we could put a comment above this block of code?

Reply:
done!